packages/platform-sdk/test/integration-tests/order/order.test.ts

```diff
@@ -185,6 +185,7 @@ describe('testing order API calls', () => {
   })
 
   it('should search a order', async () => {
+    jest.setTimeout(120000)
```

Comment:
According to the official docs the timeout is set differently: https://jestjs.io/docs/api#testname-fn-timeout